lib/elixir/lib/macro.ex

```diff
@@ -2469,20 +2469,18 @@ defmodule Macro do
   # as possible and call out into a function as soon as we can.
   @doc false
   def __dbg__(header_string, to_debug, options) do
+    {print_header?, options} = Keyword.pop(options, :print_header, true)
```

Comment:
I opted to add this to share most of the code for the non-pipe `IEx.Pry.dbg/3`.

Reply:
Should we document this option somewhere then?

Reply:
@josevalim I was inclined to keep it private API for now, but if you think it's useful, we can expose it, sure.

Reply:
I see. Private works for me, but perhaps let's rename it to `print_location` just in case?